delivery/models/project.py

```diff
@@ -154,6 +155,17 @@ def __del__(self):
                 " Check token expiry date (`dds auth info`).")
             raise
 
+    def get_dds_version(self):
+        """
+        Get the DDS version been used
+        """
+
+        result = subprocess.run(['dds', '--version'], stdout=subprocess.PIPE)
```

Comment:
For consistency with the rest of the code, you could probably use `_run` here, this will ensure that errors will be logged too.